### ISSUE TITLE
{Linked:Bug15457038}[Adaptive Cards > Designer]: NVDA is not announcing the whole information present in the Comment edit field.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9319]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/samples/InputsWithValidation.html
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Screen Reader: NVDA

**Repro Steps:**
1. Turn on NVDA..
2. Open the above URL on the Edge Dev browser.
3. Navigate to the 'Sample' control present on the header section of the page using tab key and activate using enter key.
4. Navigate to the Inputs with Validation control and activate it.
5. Press tab key and navigate to the 'Comment' edit field under Input,text element section and observe the issue.

**Actual Result:**
While navigating to the comment edit field, screen reader is not announcing the multiple line information. Here, it is announcing as **'Comment edit enter any comment test line 1.'**

**Note:**
Same issue is working fine with Narrator.

**Expected Result:**
NVDA should announce the whole information present in the Comment edit field and it should announce as **'Comment edit field test line 1 test line 2 test line 3**'

**User Experience:**
Users who rely on screen reader for navigation will be impacted if NVDA is not announcing the whole information present in the Description edit field.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/d80a6dea-49e9-402e-bcc6-1d62ad1a81b2

### Card JSON

```JSON
NA
```

### Sample Code

Originally tracking with https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/15457038
